### ISSUE TITLE
Concatenate Strings or Columns

### ISSUE THREAD

**Issue by vivekratnavel** [agershun/alasql#473]
Is there an in-built function to concatenate columns or strings like CONCAT() in mysql?

**Comment by mathiasrw** [comment]
Hmmm... dont know why `CONCAT()` is not supported. 

You can make a user defined function to let it work. 

```
alasql.fn.CONCAT = function(){
return arguments.join('');
}
```

now it should work (Sorry - cant test it now)

This needs to be implemented as part of the library

**Comment by nickdeis** [comment]
@mathiasrw 
Close, but the arguments object is not an Array, so you'd have to do something like

```javascript
alasql.fn.CONCAT = function(){
return Array.prototype.slice.call(arguments).join(' ');
}
```

**Comment by mathiasrw** [comment]
:+1:

**Comment by agershun** [comment]
Just added CONCAT() to the standard library.

**Comment by mathiasrw** [comment]
Closed per request from #628
